fdm-app/app/components/custom/soil/form.tsx

```diff
@@ -0,0 +1,412 @@
+                                                        <CalendarIcon className="ml-auto h-4 w-4 opacity-50" />
+                                                    </Button>
+                                                </FormControl>
+                                            </PopoverTrigger>
+                                            <PopoverContent
+                                                className="w-auto p-0"
+                                                align="start"
+                                            >
+                                                <Calendar
+                                                    mode="single"
+                                                    selected={field.value}
+                                                    onSelect={field.onChange}
+                                                    locale={nl}
+                                                    disabled={(date) =>
+                                                        date <
+                                                        new Date("1970-01-01")
+                                                    }
+                                                    initialFocus
+                                                />
+                                            </PopoverContent>
+                                        </Popover>
+                                        <FormDescription>
+                                            {getParameterDescription(
+                                                soilParameterDescription,
+                                                field.name,
+                                            )}
+                                        </FormDescription>
+                                        <FormMessage />
+                                    </FormItem>
+                                )}
+                            />
+                            <FormField
+                                control={form.control}
+                                name="a_p_al"
+                                render={({ field }) => (
+                                    <FormItem>
+                                        <FormLabel>
+                                            {getParameterName(
+                                                soilParameterDescription,
+                                                field.name,
+                                            )}
+                                        </FormLabel>
+                                        <FormControl>
+                                            <Input
+                                                {...field}
+                                                type="number"
+                                                value={field.value}
+                                                placeholder="Bv. 14.5 mg P2O5/kg"
+                                            />
```

Comment:
als PAL than per 100 g

Reply:
Will be resolved in 8f5c0104ec20968923e3e75e88a44e271a90827a